website/docs/sql-reference/joins/sql-self-join.md

```diff
@@ -0,0 +1,71 @@
+---
+id: self-join
+title: SQL SELF JOINS
+description: A self join allows you to join a dataset back onto itself. A common use cases to leverage a self join is when a table contains a foreign key to the primary key of that same table.
+slug: /sql-reference/self-join
+---
+
+<head>
+    <title>Working with self joins in SQL</title>
+</head>
+
+Simultaneously the easiest and most confusing of joins: the self join. Simply put, a self join allows you to join a dataset back onto itself.
+
+If you’re newer to data work or SQL, you may be asking yourself: why in the world would you want to do this? Shouldn’t joins happen between multiple *different* entities?
+
+The majority of joins you see in analytics work and dbt projects will probably be left and inner joins, but occasionally, depending on how the raw source table is built out, you’ll leverage a self join. One of the most common use cases to leverage a self join is when a table contains a foreign key to the <Term id="primary-key" /> of that same table.
+
+It’s ok if none of that made sense—jump into this page to better understand how and where you might use a self join in your analytics engineering work.
+
+## How to create a self join
+
+No funny venn diagrams here—there’s actually even no special syntax for self joins. To create a self join, you’ll use a regular join syntax, the only differences is the join objects are *the same*:
+
+```
+select
+	<fields>
+from <table_1> as t1
+[<join_type>] join <table_2> as t2
+on t1.id = t2.id
+```
+
+Since you can choose the dialect of join for a self join, you can specify if you want to do a [left](/sql-reference/left), [outer](/sql-reference/outer-join), [inner](/sql-reference/inner-join), [cross](/sql-reference/cross-join), or [right join](/sql-reference/right-join) in the join statement.
```

Comment:
```suggestion
Since you can choose the dialect of join for a self join, you can specify if you want to do a [left](/sql-reference/left-join), [outer](/sql-reference/outer-join), [inner](/sql-reference/inner-join), [cross](/sql-reference/cross-join), or [right join](/sql-reference/right-join) in the join statement.
```